File: app/src/main/java/com/grsu/guideapp/holders/object/ObjectHolder.java
package com.grsu.guideapp.holders.object;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;
import com.grsu.guideapp.R;
import com.grsu.guideapp.base.BaseViewHolder;
import com.grsu.guideapp.base.listeners.ItemClickListener;
import com.grsu.guideapp.models.DtoObject;
import com.squareup.picasso.Picasso;

public class ObjectHolder extends BaseViewHolder<DtoObject> {

//    @BindView(R.id.item_type_image)
    ImageView mItemTypeImage;
//    @BindView(R.id.item_type_name)
    TextView mItemTypeName;

    public ObjectHolder(View itemView, ItemClickListener listener) {
        super(itemView, listener);
    }

    @Override
    public void bind(DtoObject object) {
        mItemTypeName.setText(object.getName());
        Picasso.get().load(object.getPhoto()).error(R.drawable.ic_launcher_background).into(mItemTypeImage);
    }
}

File: app/src/main/java/com/grsu/guideapp/ui/holders/object/ObjectHolder.kt
package com.grsu.guideapp.ui.holders.`object`

import android.view.View
import android.widget.ImageView
import android.widget.TextView
import com.grsu.guideapp.R
import com.grsu.guideapp.utils.base.BaseViewHolder
import com.grsu.guideapp.data.local.database.vo.ObjectItemVO
import com.squareup.picasso.Picasso

class ObjectHolder(
    itemView: View,
    listener: ((View, Int) -> Unit)
) : BaseViewHolder<ObjectItemVO>(itemView, listener) {

    private val image = itemView.findViewById(R.id.item_type_image) as? ImageView
    private val name = itemView.findViewById(R.id.item_type_name) as? TextView

    override fun bind(item: ObjectItemVO) {
        name?.text = item.name
        val creator = Picasso.get().load(item.image)
            .placeholder(R.drawable.ic_launcher_foreground)
            .error(R.drawable.ic_launcher_foreground)
        if (image != null) creator.into(image)
    }
}